techniques/aria/ARIA11.html

```diff
@@ -26,7 +26,7 @@ <h1>Using ARIA landmarks to identify regions of a page</h1>
          <li><code class="language-html">navigation</code>: A region that contains navigation links links to other pages or different parts of the same page.</li>
          <li><code class="language-html">main</code>: A region that contains a page's main content.</li>
          <li><code class="language-html">region</code>: A region that contains a perceivable section of the page containing content that is sufficiently important for users to be able to navigate to the section. A <code class="language-html">region</code> landmark isn't exposed as a landmark region unless it has an accessible name.</li>
-         <li><code class="language-html">form</code>: A region of the document that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing.</li>
+         <li><code class="language-html">form</code>: A region of the document that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing. It maps to <code class="language-html">form</code> in HTML, and needs an accessible name to be exposed as a landmark.</li>
```

Comment:
not against the idea of mentioning what element this maps to in html - but now this is the only one that does.

there are a bunch of other nits i have with this doc (outside of the scope of this PR) - would it be ok to rectify those with this PR, or do we need to merge this and then i make another PR to make those corrections?

Reply:
Depends! I'm increasingly not a fan of PRs that wander off topic, but if it's just a couple of minor things then we could add them. I could also back out the mapping content to put in another PR if your bunch of nits is large.